### PR TITLE
update userdomain module

### PR DESCRIPTION
**Pull request by cgzones**
* grant admins capability2:wake_alarm
* remove old /run/user file context pattern

### CODE REVIEW

File: policy/modules/system/userdomain.if [hunk 1]
```diff
@@ -1168,6 +1168,7 @@ template(`userdom_admin_user_template',`
 	#
 
 	allow $1_t self:capability ~{ sys_module audit_control audit_write };
+	allow $1_t self:capability2 wake_alarm;
```
Comment (pebenito) [inline policy/modules/system/userdomain.if]: What apps trigger this?

Comment (cgzones) [inline policy/modules/system/userdomain.if]: a self written dyndns updater
```
type=PROCTITLE msg=audit(02/16/17 17:10:17.643:885) : proctitle=dtdnssync check 
type=SYSCALL msg=audit(02/16/17 17:10:17.643:885) : arch=armeb syscall=timerfd_create per=PER_LINUX_32BIT success=yes exit=5 a0=0x1 a1=0x80000 a2=0x0 a3=0x1 items=0 ppid=25279 pid=26679 auid=christian uid=root gid=root euid=root suid=root fsuid=root egid=root sgid=root fsgid=root tty=pts0 ses=71 comm=dtdnssync exe=/usr/sbin/dtdnssync subj=staff_u:sysadm_r:sysadm_t:s0 key=(null) 
type=AVC msg=audit(02/16/17 17:10:17.643:885) : avc:  granted  { wake_alarm } for  pid=26679 comm=dtdnssync capability=wake_alarm  scontext=staff_u:sysadm_r:sysadm_t:s0 tcontext=staff_u:sysadm_r:sysadm_t:s0 tclass=capability2
```

maybe related to https://github.com/SELinuxProject/selinux-kernel/issues/28?

File: policy/modules/system/userdomain.if [hunk 2]
```diff
@@ -1167,7 +1167,8 @@ template(`userdom_admin_user_template',`
 	# $1_t local policy
 	#
 
-	allow $1_t self:capability ~{ audit_control audit_write sys_module };
+	allow $1_t self:capability ~{ sys_module audit_control audit_write };
```
Comment (pebenito) [inline policy/modules/system/userdomain.if]: Please drop this line, since it will undo a bit of 1720e109a325f0be8aeae54e2e50d5cd2498629b.

**Comment by cgzones** [comment]
I'll check back after testing with https://github.com/SELinuxProject/selinux-kernel/issues/28 fix

**Pull request closed without merging**